CHANGELOG.md

```diff
@@ -1,68 +1,108 @@
-# Changelog
+# CB Event Forwarder Changelog
 
-## cb-event-forwarer 3.5.0
-- kafka SASL support
-- OATH2 JWT optional support for http output
-- Support for sending EventText as bytearary httpoutput 
+## v3.6.2
 
-## cb-event-forwarder 3.1.2
+#### Features
+
+ * Event Forwarder can now be configured and operated from the CB EDR web console.
+ * There are no new features in Event Forwarder itself.
+
+#### Bug Fixes
+
+ * Fix signal handling for syslog and S3 output types
+ * Fix error handling for AMQP connections
+
+## v3.6.1
+
+### Features
+
+ * CentOS/RHEL 7.x compatibility with separate packages for el6 and el7.
+ * New metric support
+ * Threading for Kafka output,
+ * bility to configure more options for kafka.
```

Comment:
Copy pasta issue.  Should read Ability, vice bility.

Reply:
Thanks, fixed.